suite/tests/client-interface/low_on_memory.c

```diff
@@ -41,21 +8,29 @@
 #else
 #    include <stdio.h>
 #    include <stdlib.h>
-#    include <string.h>
 #endif
 
 int
 main(int argc, char **argv)
 {
-
+#ifdef WINDOWS
+    HANDLE heap = GetProcessHeap();
+#endif
     int total = 0;
     for (int i = 0; i < 200; i++) {
-
+#ifdef WINDOWS
+        int *my_integer = (int *)HeapAlloc(heap, HEAP_ZERO_MEMORY, sizeof(int));
+#else
         int *my_integer = (int *)malloc(sizeof(int));
+#endif
         *my_integer = 9;
         total += *my_integer;
+#ifdef WINDOWS
+        HeapFree(heap, 0, my_integer);
+#else
         free(my_integer);
+#endif
     }
 
-    printf("My total is %d\n", total);
+    fprintf(stderr, "My total is %d\n", total);
```

Comment:
As mentioned, please use print() from tools.h.  That will do the proper flushing on Windows.

Reply:
Done